spec/spec_helper.rb

```diff
@@ -5,8 +5,10 @@
   minimum_coverage 97
 end
 
-Dir['./spec/support/**/*'].each do |f|
-  require f.sub(%r|\./spec/|, '')
+Dir.chdir('spec') do
```

Comment:
Why?

Reply:
I've split shared examples into several files and placed them under `spec/support/shared_examples` directory. `Dir` matched both files and the directory itself.
I couldn't resist the temptation for this change but will minimize it so `Dir` only yields `.rb` files